storage/migrations/22_consensus_accounts_first_activity.up.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+
+ALTER TABLE chain.accounts
+	ADD COLUMN first_activity TIMESTAMP WITH TIME ZONE;
```

Comment:
Would it work to store the `first_activity_block` instead and `join` it with the blocks table for the timestamp?

Reply:
I almost had it done like that already but it doesnt work nicely with the static accounts. Since static accounts would need to insert fist activity at block height that wont even exist in the indexer db.

Reply:
Ah right, ok that makes sense